utils/systemd/monerod.service

```diff
@@ -6,12 +6,13 @@ After=network.target
 User=monero
 Group=monero
 WorkingDirectory=~
+RuntimeDirectory=monero
```

Comment:
The man page (systemd.exec(5)) says this is created in /run, nor /var/run. It seems like I have a /run here, and /var/run is a symlink to this, unexpectedly. Is this "standard" across systemd, or user selected ?

Reply:
Good point. Let's stick to systemd's /run and not rely on the link to be present. Tested and pushed.